components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/test/java/org/wso2/carbon/identity/api/resource/mgt/dao/APIResourceManagementDAOImplTest.java

```diff
@@ -77,18 +84,27 @@ public void tearDown() throws Exception {
     @DataProvider
     public Object[][] getAPIResourcesCountData() {
         return new Object[][]{
-                {TENANT_ID, new ArrayList<>(), 2},
-                {INVALID_TENANT_ID, new ArrayList<>(), 0},
+                {TENANT_ID, new ArrayList<>(), false, 2},
+                {INVALID_TENANT_ID, new ArrayList<>(), true, 0},
```

Comment:
Here the combinations are less.

We need to execute the test for correct TENANT_ID and INVALID_TENANT_ID when isOrganization is false
TENANT_ID and INVALID_TENANT_ID when isOrganization is true

Reply:
Do we need to test the INVALID_TENANT_ID when isOrganization is true?

Reply:
Added a new combination